Python/Tests/VSInterpretersTests/PipRequirementsUtilsTests.cs

```diff
@@ -215,6 +181,30 @@ public class PipRequirementsUtilsTests {
                 new[] { new PackageSpec("Django", "1.8") }
             ));
 
+            Assert.IsFalse(PipRequirementsUtils.AnyPackageMissing(
+                new[] { "django >= 2.1.5,<3.0 " },
+                new[] { new PackageSpec("Django", "1.8") }
+            ));
+
+            Assert.IsFalse(PipRequirementsUtils.AnyPackageMissing(
+                new[] { "django >= 2.1.5, <= 3.0 " },
+                new[] { new PackageSpec("Django", "2.2") }
+            ));
+            Assert.IsFalse(PipRequirementsUtils.AnyPackageMissing(
+                new[] { "django>=2.1.5,<=3.0#comment" },
+                new[] { new PackageSpec("Django", "2.2") }
+            ));
```

Comment:
please add IsFalse tests for these:
```
-r morereqs.txt
git+https://myvcs.com/some_dependency
git+https://myvcs.com/some_dependency@sometag#egg=SomeDependency
```

Reply:
Since the "AnyPackagesMissing" cannot parse those 3 cases, shouldn't this be "Assert.IsTrue()"?

Reply:
No. We don't want to prompt the user that packages are missing if they have those things in their requirements.txt.

Reply:
Will be updated shortly.